brainiak/utils/utils.py

```diff
@@ -18,6 +18,7 @@
 import psutil
 from .fmrisim import generate_stimfunction, _double_gamma_hrf, convolve_hrf
 from scipy.fftpack import fft, ifft
+
```

Comment:
Please remove the whitespace changes in this file.

Reply:
Oops, another good catch -- thanks. Fixed.